### PR TITLE
fix CommonVoice prepare

### PR DESCRIPTION
**Pull request by mohsen-goodarzi**
This PR will solve the prepare step of recent versions of CommonVoice mentioned here #740.
In recent CommonVoice, they changed the `accent `column to `accents`.
This solution should be backward compatible, but the corresponding field in supervision json will be `accents `instead of `accent`.

### CODE REVIEW

File: lhotse/recipes/commonvoice.py [hunk 1]
```diff
@@ -263,6 +263,11 @@ def parse_utterance(
         raise ValueError(f"No such file: {audio_path}")
     recording_id = Path(row.path).stem
     recording = Recording.from_file(audio_path, recording_id=recording_id)
+    # Handling accent(s) in different versions of CommonVoice
+    if "accents" in row:
```
Comment (csukuangfj) [inline lhotse/recipes/commonvoice.py:267]: I usually see people using `hasattr(row, 'accents')`.
The operator `in` is commonly used together with `list`, `set` and`dict`.

Not sure whether using the operator `in` is appropriate here.

Comment (pzelasko) [reply lhotse/recipes/commonvoice.py:267]: AFAIK `in` checks for column existence in pandas objects, so it looks OK to me.

Comment (pzelasko) [reply lhotse/recipes/commonvoice.py:267]: Actually `hasattr` might not work, I think pandas "columns-as-members" thing works only dynamically via `getattr` (unless they explicitly added support for `hasattr`)

Comment (csukuangfj) [reply lhotse/recipes/commonvoice.py:267]: Ah, thanks. I was not aware that it is a pandas object.

**Pull request merged**